docs/api/api.md

```diff
@@ -1,343 +1,253 @@
 # Full API list
-<!--these are now just the APIs - do I want to add the tutorials?-->
+
```

Comment:
I am assuming that this page is replaced by the alternative.

Reply:
Yes